va/va_vpp.h

```diff
@@ -1313,8 +1333,13 @@ typedef struct _VAProcFilterParameterBufferHVSNoiseReduction {
      *  Value 10 is the default value.
      */
     uint16_t            strength;
+    /**
+     *  \brief HVS Denoise Mode which controls denoise method.
+     *  It is a value in VAProcHVSDenoiseMode.
+     */
+    uint16_t            mode;
```

Comment:
@XinfengZhang : what's our approach to specify integer type for enum value?

Reply:
Yeah, but if we will have `VAProcHVSDenoiseMode mode`, what will be its size? Generally speaking it's compiler dependent (but I again might be over paranoid).

Reply:
it will be 32bit on both 64bit and 32 bit compiler. but your concern make sense. maybe we could use macro instead of enum to avoid such risk.